wesnoth-tb/bn.po

```diff
@@ -0,0 +1,1293 @@
+msgstr "আচ্ছা, এলভেস?এখন আপনি দেখতে পাচ্ছেন যে নেক্রোম্যান্সাররা কে?"
+
+#. [message]: speaker=messenger_elf
+#. a elven scout rides in from off-map, takes their first look at the
+#. situation, and realises that they have no idea what's going on
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:517
+msgid "I heard sounds of battle, but did not expect to see this. I will inform the Council, they can work out who the necromancers are."
+msgstr "আমি যুদ্ধের শব্দ শুনেছি, তবে এটি দেখার আশা করি নি।আমি কাউন্সিলকে অবহিত করব, তারা নেক্রোম্যান্সাররা কারা তা নিয়ে কাজ করতে পারে।"
+
+#. [message]: speaker=Arvith
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:544
+msgid "Come on, men, let’s catch those kidnappers!"
+msgstr "আসুন, পুরুষরা, আসুন সেই অপহরণকারীদের ধরুন!"
+
+#. [message]: speaker=unit
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:556
+msgid "Hah! You have captured me, but it will avail you nothing. I sent your precious brother the mage north with half my men a day since; he will be safely locked away in our master’s dungeons by now."
+msgstr "হাহ!আপনি আমাকে বন্দী করেছেন, তবে এটি আপনাকে কিছুই উপকার দেবে না।আমি আপনার মূল্যবান ভাইকে আমার অর্ধেক পুরুষদের সাথে একদিনের পরে প্রেরণ করেছি;এখনই তিনি নিরাপদে আমাদের মাস্টারের অন্ধকূপে লক হয়ে যাবেন।"
+
+#. [message]: speaker=Arvith
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:561
+msgid "My blade is at your throat. Give us the way to my brother now, or I will spill your wretched blood on the ground."
+msgstr "আমার ব্লেড আপনার গলায়।আমাদের এখনই আমার ভাইয়ের পথ দিন, বা আমি আপনার খারাপ রক্ত মাটিতে ছড়িয়ে দেব।"
+
+#. [event]
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:564
+msgid "Sithrak"
+msgstr "সিঙ্গার"
+
+#. [event]
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:565
+msgid "Eleben"
+msgstr "রিংলিডি"
+
+#. [event]
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:566
+msgid "Jarlom"
+msgstr "আইলুম"
+
+#. [event]
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:567
+msgid "Hamik"
+msgstr "হামিক"
+
+#. [event]
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:571
+msgid "Akranbral"
+msgstr "আখারামব্রাল"
+
+#. [event]
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:572
+msgid "Drakanal"
+msgstr "অন্ধকার"
+
+#. [event]
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:573
+msgid "Xaskanat"
+msgstr "র‌্যাকটিয়ার"
+
+#. [event]
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:574
+msgid "Katklagad"
+msgstr "ক্যাটল্যাকস"
+
+#. [message]: speaker=unit
+#: data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg:580
+msgid "Three days ride to the northeast, in a deserted castle. The passwords to the guards are $first_password_$first_password and $second_password_$second_password|."
+msgstr "তিন দিন উত্তর -পূর্ব দিকে যাত্রা করে একটি নির্জন দুর্গে।গার্ডদের পাসওয়ার্ডগুলি হ'ল $ প্রথম_প্যাসওয়ার্ড_ $ প্রথম_প্যাসওয়ার্ড এবং $ সেকেন্ড_প্যাসওয়ার্ড_ $ সেকেন্ড_প্যাসওয়ার্ড |"
```

Comment:
This one contains WML variables - sometime in the next 24 hours, documentation about how it should be translated will be added to <https://wiki.wesnoth.org/GettextForTranslators>, but it isn't there yet.

Reply:
Will be cleaned up.

Reply:
fixed in ae153a789173dceec199454650b4d83022775829